### ISSUE TITLE
Final Peer Review: arb379

### ISSUE THREAD

**Issue by AbbyBeeler** [kew96/LoanMachineLearning#16]
This project is about predicting the correct interest loan rates for customers interested in one. The main data transformation was from categorical to numerical using label encoding, one hot encoding, and an offset. To get an initial baseline, they used linear regression with the eight most important features to get a mean average error of 1.19. The next discussed model used ridge regression with gaussian noise specifically because it handles multicollinearity, penalizes outliers heavily, and it doesn't remove any features to get a MAE of 0.78801. The final model used XGBoost with hyperparameter search to get a MAE 0.41694. 

While the project in nature has the potential to be a weapon of mass destruction as it influences people's lives, this group has done what they can to mitigate the risk by using lots of data and multiple models. They also analyzed fairness on the basis of location (the only applicable characteristic in the dataset). Some states have slightly different distributions, but roughly fair. Location and job position information was not included in model. In the future, the group would like to further reduce bias by examining more info about protected characteristics, build a model to classify bad and good loans, and getting more computational power to add more training for rounds in XGBoost and hyperparameter search. 

Things I Liked
 - I liked that they picked a really simple model as a baseline for comparison. It was an objective way to determine progress. 
 - It was a good idea to use XGBoost as it dramatically decreased error. There was also a good explanation of what it did. 
 - There was a very well-reasoned discussion about whether this model was a WMD. It recognized the potential, but also described what they had done to prevent it. 

Areas of Improvement 
 - The report could have used a more thorough explanation of how missing data was filled using expected values as well as a more detailed reasoning of how they knew those 112 features were irrelevant. 
 - It would have been nice to have an explanation of how the packaged used for feature importance worked. 
 - Interest on a loan seems rather subjective and there wasn't any discussion of whether the given loan amounts were "correct".
